<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="gmailSuite" duration-ms="4330" started-at="2015-11-29T02:52:09Z" finished-at="2015-11-29T02:52:14Z">
    <groups>
    </groups>
    <test name="registerTest" duration-ms="4330" started-at="2015-11-29T02:52:09Z" finished-at="2015-11-29T02:52:14Z">
      <class name="com.test.GmailRegistrationTest">
        <test-method status="FAIL" signature="testValidateform()[pri:0, instance:com.test.GmailRegistrationTest@1bc6a36e]" name="testValidateform" duration-ms="4299" started-at="2015-11-28T18:52:09Z" finished-at="2015-11-28T18:52:14Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<You can't leave this empty.> but was:<>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<You can't leave this empty.> but was:<>
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
	at com.test.GmailRegistrationTest.testValidateform(GmailRegistrationTest.java:15)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testValidateform -->
      </class> <!-- com.test.GmailRegistrationTest -->
    </test> <!-- registerTest -->
  </suite> <!-- gmailSuite -->
</testng-results>
